scripts/globals/items/caliber_ring.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- ID: 26164
+-- Item: caliber ring
+-- Experience point bonus
+-----------------------------------
+-- Bonus: +150%
+-- Duration: 720 min
+-- Max bonus: 30000 exp
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/item_utils")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    local result = 0
+    if target:hasStatusEffect(xi.effect.DEDICATION) then
+        result = 56
```

Comment:
Is this the same as `xi.msg.basic.ITEM_UNABLE_TO_USE_2` which is 56 from msg.lua?

Reply:
Based on the behavior of the other scripts, I'd say yes. Prefer to have the ENUM instead of the Magic Number?

Reply:
yeah, enum is preferred over magic numbers. thanks!

Reply:
Can do, update coming in a moment!